apriltag_ros/msg/Point2D.msg

```diff
@@ -0,0 +1,3 @@
+# This contains the position of a point in 2D pixel coordinates
+float64 u
+float64 v
```

Comment:
shouldn't these be `uint16_t` or `size_t`?

Reply:
The library output is in floats not integer values. Example: 
```yaml
    tag_center: 
      u: 1167.1968160351246
      v: 435.0856950109363
    tag_corners: 
      - 
        u: 1196.1888427734375
        v: 399.94015502929693
      - 
        u: 1132.5440673828127
        v: 406.6687927246094
      - 
        u: 1139.0986328125002
        v: 469.14767456054693
      - 
        u: 1201.446044921875
        v: 463.17169189453136
```